src/actions/ui-state-actions.js

```diff
@@ -95,6 +95,18 @@ export const addNotification = createAction(
   notification => notification
 );
 
+/**
+ * Updates a notification with identical id or adds a new notification to be displayed
+ * @memberof uiStateActions
+ * @param notification - The `notification` object to be updated or added
+ * @type {typeof import('./ui-state-actions').setNotification}
+ * @public
+ */
+export const setNotification = createAction(
```

Comment:
we already have a way to add notifications, can you use addNotification action?

Reply:
_onError callback from deck.gl can be called with same error message on each draw. 
addNotification adds a new notification to the end of notifications array. 
With setNotification it is possible to create or update existing notification if a notification with such id already exists.

- we don't want to show a ton of same notifications
- we don't want notifications to change order
- we want a notification to appear again if closed, if the error happens again

Reply:
can you modify the addNOtification to have the same behavior as your new setNotification? It will update a notification if you pass a new object with an existing ID so you don't have to create a new action